query.go

```diff
@@ -37,6 +37,7 @@ func (fb *Firebase) EndAt(value string) *Firebase {
 func (fb *Firebase) OrderBy(value string) *Firebase {
 	c := fb.copy()
 	if value != "" {
+		value = "\"" + value + "\""
```

Comment:
also, it looks like we're just preparing `value` before it gets passed to `c.params.Set()` on the next line. We might just do the preparation inline:

```go
c.params.Set(orderByParam, fmt.Sprintf("%q", value))
```